documentation/Demo 2/Requirments.md

```diff
@@ -0,0 +1,51 @@
+# Red Hat Beyond 07
+
+Statement of Work (SOW)
+***
+
+
+## Milestones
+
+### Demo 2 (April 3rd): Data model and business logic
+
+The purpose of this demo is to show your application's data model and data
```

Comment:
one more problem when you copy the content is that you have to make sure all the time nothing is updated on the source page

Reply:
done